DFC.ServiceTaxonomy.PageLocation/Handlers/PageLocationTaxonomyTermHandler.cs

```diff
@@ -22,23 +22,34 @@ public class PageLocationTaxonomyTermHandler : ITaxonomyTermHandler
         private readonly ITaxonomyHelper _taxonomyHelper;
         private readonly ISyncOrchestrator _syncOrchestrator;
         private readonly IContentItemsService _contentItemsService;
+        private readonly IEnumerable<IContentOrchestrationHandler> _contentOrchestrationHandlers;
 
         public PageLocationTaxonomyTermHandler(
             ISession session,
             ITaxonomyHelper taxonomyHelper,
             ISyncOrchestrator syncOrchestrator,
-            IContentItemsService contentItemsService)
+            IContentItemsService contentItemsService,
+            IEnumerable<IContentOrchestrationHandler> contentOrchestrationHandlers)
         {
             _session = session;
             _taxonomyHelper = taxonomyHelper;
             _syncOrchestrator = syncOrchestrator;
             _contentItemsService = contentItemsService;
+            _contentOrchestrationHandlers = contentOrchestrationHandlers;
         }
 
-        public async Task UpdatedAsync(ContentItem term, ContentItem taxonomy)
+        public async Task PublishedAsync(ContentItem term)
+        {
+           foreach(var orchestrator in _contentOrchestrationHandlers)
+            {
+                await orchestrator.Published(term);
+            }
+        }
+
+        public async Task<bool> UpdatedAsync(ContentItem term, ContentItem taxonomy)
         {
             if (term.ContentType != ContentTypes.PageLocation)
-                return;
+                return false;
```

Comment:
This is a check to only apply the handler's logic for `PageLocation`'s and not all taxonomy terms.
So if this handler doesn't apply, it has been successful and should return true (so that it's a no-op for other taxonomy types, rather than an error).
We also need the same check in `PublishedAsync`, so that it only applies to `PageLocation`'s.